.github/workflows/dependent-tests.yml

```diff
@@ -26,9 +26,10 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
+      # TODO: change target before merging into public
       run: |
         pip install --upgrade pip
-        pip install --upgrade git+https://github.com/aws/amazon-braket-schemas-python@main
+        pip install --upgrade git+https://github.com/aws/amazon-braket-schemas-python@feature/openqasm-local-simulator
```

Comment:
Do you want to merge this to main?

Reply:
This is the PR into the feature branch. The line of code highlighted here points the dependent tests to the schema feature branch, which is where the schema changes will stay until we're ready to hard launch